client/src/components/body/landingpage/index.js

```diff
@@ -0,0 +1,261 @@
+                  >
+                </h4>
+                <p class="">
+                  Learn more about Screen Readers.
+                </p>
+              </div>
+            </div>
+            <div
+             class="col-md-4 offset-md-4 portfolio-item">
+              <a
+                class="portfolio-link"
+
+              >
+                <img
+                  class="img-fluid landingpage__image"
+                  src={hand}
+                  alt="Dexterity activity Thumbnail"
+                />
+                {/* --Image= https://www.needpix.com/photo/1179306/hand-finger-symbol-gesture-characters-note-show-exhort-suggest-- */}
+              </a>
+              <div class="portfolio-caption">
+                <h4>
+                  <a
+                    >Accessibility to Dexterity</a
+                  >
+                </h4>
+                <p class="">
+                  Learn more about designing the web for individuals with motor and dexterity impairments.
+                </p>
+              </div>
+            </div>
+            </div>
+      </div>
+    </section>
+    <hr class="horiz" />
+    {/* Clients */}
+    <section class="py-5">
+      <div class="container">
+        <div
+          class="row landingpage__logos"
+        >
+          <div class="col-sm-4">
+            <a href="https://www.nsf.gov" target="_blank">
+              <img
+                class="d-block mx-auto"
+                src={nsf}
+                alt="National Science Foundation"
+                height="100"
+              />
+            </a>
+          </div>
+          <div class="col-sm-4 landingpage__ritlogo">
+            <a href="https://www.rit.edu" target="_blank">
+              <img
+                class=" d-block mx-auto"
+                src={rit}
+                alt="Rochester Institute Of Technology"
+                height="100"
+              />
+            </a>
+          </div>
+        </div>
+      </div>
+    </section>
+    {/* Contact */}
+    <section class="page-section" id="contact">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-12 text-center">
+            <h2 class="section-heading text-uppercase">Contact Us</h2>
+            <h3 class="section-subheading text-primary">
+              Connect with us if you have questions. Contact Dr. Daniel Krutz at
+              <a class="landingpage__contact" href="mailto:aware@mail.rit.edu "> aware@mail.rit.edu</a>
+            </h3>
+          </div>
+        </div>
+      </div>
+    </section>
+    {/* Footer */}
+    <footer class="footer">
+      <ul class="list-inline quicklinks">
+        <li class="list-inline-item">
+          <a
+            href="https://www.nsf.gov/pubs/2016/nsf16009/nsf16009.jsp#q37" target="_blank"
+            >Available under the Federal Government License</a
+          >
+        </li>
+        <li>
+          <a href="sitemap.js" >
```

Comment:
should have the sitemap component called like <SiteMap /> and import at the top. Refer to app.js to see how we do it to add the header component